spec/dependency_worker_spec.rb

```diff
@@ -91,6 +105,7 @@ def to_s
     }
     
     context 'when the dependencies have changed' do
+      let(:dependency_diff) {  {'foobar' => :bad_change, 'gollum-lib' => :bad_change, 'kramdown-parser-gfm' => :bad_change} }
```

Comment:
Shouldn't we have at least one good_change in the tests?

Reply:
Good idea! Required using different fixtures, but worth it.